engine/src/main/java/com/google/android/fhir/FhirEngine.kt

```diff
@@ -34,6 +34,12 @@ interface FhirEngine {
    */
   suspend fun create(vararg resource: Resource): List<String>
 
+  /**
+   * Creates one or more remote FHIR [resource]s in the local storage without flagging as local
+   * changes.
+   */
+  suspend fun createRemote(vararg resource: Resource)
+
```

Comment:
```suggestion
  /**
   * Creates one or more FHIR [resource]s in the local storage.
   *
   * @param isLocalOnly - Setting the value to [true] instructs engine that the resource and its subsequent updates should never be synced to the server.
   * @return the logical IDs of the newly created resources.
   */
    suspend fun create(vararg resource: Resource, isLocalOnly: Boolean = false): List<String> 
```

The default value can be provided in the interface i.e [FhirEngine.kt](https://github.com/google/android-fhir/pull/2178/files#diff-c31e3e53f14056cbc724b023efe0b83d19a7d82d9c9332197e1bb1bd99e08bb1) and the [FhirEngineImpl.kt](https://github.com/google/android-fhir/pull/2178/files#diff-5b081449c3e8ee8c1459f4f7b52966b1a63fe92d79beaee1b8ed0a4b77a14626) would be called with that value if no value is provided by the caller.